internal/db/base/compare.go

```diff
@@ -39,6 +39,8 @@ func Compare(a, b any) int {
 		return compareInt(v, b.(int64))
 	case uint64:
 		return compareUint(v, b.(uint64))
+	case float32:
```

Comment:
todo: Please either delete or cover this with an integration test (assuming code-cov isn't faulty, maybe add a panic and see if any tests fail).

Reply:
Done